src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkWildcardsInspectionVisitor.kt

```diff
@@ -0,0 +1,23 @@
+package com.jetbrains.snakecharm.inspections
+
+import com.intellij.codeInspection.LocalInspectionToolSession
+import com.intellij.codeInspection.ProblemsHolder
+import com.jetbrains.snakecharm.lang.psi.SmkRuleOrCheckpoint
+
+abstract class SmkWildcardsInspectionVisitor<T>(
```

Comment:
I prefer to name abstract classes with 'Abstract' prefix or use `Base` suffix, much easier to find it in future and to filter it out from other results in `goto to class` action results. E.g. here  `AbstractSmkWildcardsInspectionVisitor`